main_static.go

```diff
@@ -0,0 +1,17 @@
+//go:build linux
+// +build linux
+
+package main
+
+import (
+	_ "time/tzdata"
+
+	_ "github.com/breml/rootcerts"
+
+	reaper "github.com/ramr/go-reaper"
```

Comment:
Did not know this tool, looks useful. Theoretically, we don't need any init processes, because blocky doesn't spawn any chlild processes. Tini was introduced by a PR to have some kind of a "safety net". Docker also includes "-init" functionality for running containers.

I think we can try go-reaper, I hope it is stable and has no impact at runtime

Reply:
It is good practice to have some init process with reaper functionality.
If i remember correctly i participated in the discussion around the introduction of tini. 🤔😅

I also currently use go-reaper as some kind of "safety net" in allmost all my go containers(if they are based on scratch images) without complications so far.